frameworks/kafka/tests/test_jmx.py

```diff
@@ -0,0 +1,204 @@
+                         }
+                        }, "brokers": {"cpus": 0.5, "count": 1}}
+
+    if self_signed_trust_store:
+        service_options = sdk_utils.merge_dictionaries(
+            {"service": {"jmx": {"add_trust_store": True,
+                                 "trust_store": "/test/integration/kafka/keystore",
+                                 "trust_store_password_file": "/test/integration/kafka/keypass"}}},
+            service_options)
+    if sdk_utils.is_strict_mode():
+        kafka_sa = "test/integration/kafka/service-account"
+        kafka_sa_secret = "test/integration/kafka/secret"
+
+        sdk_security.setup_security(
+            config.SERVICE_NAME,
+            service_account=kafka_sa,
+            service_account_secret=kafka_sa_secret,
+        )
+        service_options = sdk_utils.merge_dictionaries(
+            {"service": {"service_account": kafka_sa, "service_account_secret": kafka_sa_secret}},
+            service_options
+        )
+
+    sdk_install.install(
+        package_name=config.PACKAGE_NAME,
+        service_name=foldered_name,
+        additional_options=service_options,
+        expected_running_tasks=1
+    )
+
+
+def uninstall_jmx_secrets():
+    sdk_security.delete_secret("/test/integration/kafka/keystore")
+    sdk_security.delete_secret("/test/integration/kafka/passwordfile")
+    sdk_security.delete_secret("/test/integration/kafka/access")
+    sdk_security.delete_secret("/test/integration/kafka/keypass")
+
+
+@pytest.mark.sanity
+def test_secure_jmx_configuration():
+    foldered_name = sdk_utils.get_foldered_name(config.SERVICE_NAME)
+
+    try:
+        install_jmx_configured_kafka()
+        broker_task_id_0 = sdk_tasks.get_task_ids(foldered_name)[0]
+        install_jmxterm(task_id=broker_task_id_0)
+        generate_jmx_command_files(task_id=broker_task_id_0)
+
+        cmd = "export JAVA_HOME=$(ls -d ${MESOS_SANDBOX}/jdk*/jre) && " \
+              "${JAVA_HOME}/bin/java " \
+              "-Duser.home=${MESOS_SANDBOX} " \
+              "-Djdk.tls.client.protocols=TLSv1.2 -Djavax.net.ssl.trustStore=${MESOS_SANDBOX}/jmx/trust_store " \
+              "-Djavax.net.ssl.trustStorePassword=deleteme " \
+              "-Djavax.net.ssl.keyStore=${MESOS_SANDBOX}/jmx/key_store -Djavax.net.ssl.keyStorePassword=deleteme " \
+              "-Djavax.net.ssl.trustStoreType=JKS -Djavax.net.ssl.keyStoreType=JKS -jar jmxterm-1.0.1-uber.jar " \
+              "-l service:jmx:rmi:///jndi/rmi://${MESOS_CONTAINER_IP}:31199/jmxrmi -u admin -p adminpassword " \
+              "-s -v silent -n < jmx_beans_command.txt"
+
+        full_cmd = "bash -c '{}'".format(cmd)
+
+        _, output, _ = sdk_cmd.run_cli("task exec {} {}".format(broker_task_id_0, full_cmd), print_output=True)
+
+        assert "kafka.server:type=kafka-metrics-count" in output
+        assert "kafka.server:name=BrokerState,type=KafkaServer" in output
+
+        cmd = "export JAVA_HOME=$(ls -d ${MESOS_SANDBOX}/jdk*/jre) && " \
```

Comment:
I couldn't tell the difference at first without doing a diff b/w the two cmds. 

Would you mind either:
1. renaming the two `cmd`s to be more specific to beans & domain or 
2. extracting the identical parts of the command to its own variable and appending the `_command.txt` file when you create the `full_cmd` string?

Reply:
extracted the last part to `input_jmx_commands`  and also merged both tests using `pytest.mark.parametrize`